docs/usage/install/underlay/get-started-host-device-zh_CN.md

```diff
@@ -0,0 +1,159 @@
+
+      ```shell
+      ~# IFACER_INTERFACE="<NAME>"
+      ~# cat > /etc/NetworkManager/conf.d/spidernet.conf <<EOF
+      [keyfile]
+      unmanaged-devices=interface-name:^veth*;interface-name:${IFACER_INTERFACE}
+      EOF
+      ~# systemctl restart NetworkManager
+      ```
+
+## 安装 Spiderpool
+
+1. 安装 Spiderpool。
+
+    ```bash
+    helm repo add spiderpool https://spidernet-io.github.io/spiderpool
+    helm repo update spiderpool
+    helm install spiderpool spiderpool/spiderpool --namespace kube-system --set multus.multusCNI.defaultCniCRName="host-device-conf" --set plugins.installHostDeviceCNI=true
+    ```
+
+    > 如果未安装 host-device-cni, 可以通过 Helm 参数 '-set plugins.installHostDeviceCNI=true' 安装它。
+    >
+    > 如果您是国内用户，可以指定参数 `--set global.imageRegistryOverride=ghcr.m.daocloud.io` 以帮助您快速的拉取镜像。
+    >
+    > 通过 `multus.multusCNI.defaultCniCRName` 指定 multus 默认使用的 CNI 的 NetworkAttachmentDefinition 实例名。如果 `multus.multusCNI.defaultCniCRName` 选项不为空，则安装后会自动生成一个数据为空的 NetworkAttachmentDefinition 对应实例。如果 `multus.multusCNI.defaultCniCRName` 选项为空，会尝试通过 /etc/cni/net.d 目录下的第一个 CNI 配置来创建对应的 NetworkAttachmentDefinition 实例，否则会自动生成一个名为 `default` 的 NetworkAttachmentDefinition 实例，以完成 multus 的安装。
+
+2. 创建 SpiderIPPool 实例。
+
+    Pod 会从该 IP 池中获取 IP，进行 Underlay 的网络通讯，所以该 IP 池的子网需要与接入的 Underlay 子网对应。以下是创建相关的 SpiderIPPool 示例：
+
+    ```shell
+    cat <<EOF | kubectl apply -f -
+    apiVersion: spiderpool.spidernet.io/v2beta1
+    kind: SpiderIPPool
+    metadata:
+      name: ippool-test
+    spec:
+      ips:
+      - "172.18.30.131-172.18.30.140"
+      subnet: 172.18.0.0/16
+      gateway: 172.18.0.1
+      multusName: 
+      - kube-system/host-device-conf
+    EOF
+    ```
+
+3. 验证安装：
+
+    ```bash
+    ~# kubectl get po -n kube-system |grep spiderpool
+    spiderpool-agent-7hhkz                   1/1     Running     0              13m
+    spiderpool-agent-kxf27                   1/1     Running     0              13m
+    spiderpool-controller-76798dbb68-xnktr   1/1     Running     0              13m
+    spiderpool-init                          0/1     Completed   0              13m
+
+    ~# kubectl get sp ippool-test       
+    NAME          VERSION   SUBNET          ALLOCATED-IP-COUNT   TOTAL-IP-COUNT   DEFAULT
+    ippool-test   4         172.18.0.0/16   0                    10               false
+    ~# 
+    ```
+
+## 创建应用
+
+以下的示例 Yaml 中， 会创建 2 个副本的 Deployment，其中：
+
+* `v1.multus-cni.io/default-network`：用于指定 Multus 的 NetworkAttachmentDefinition 配置，会基于它为应用创建一张默认网卡。
+
+```shell
+cat <<EOF | kubectl create -f -
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-app
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: test-app
+  template:
+    metadata:
+      annotations:
+        ipam.spidernet.io/ippool: |-
+          {
+            "ipv4": ["ippool-test"]
+          }
+        v1.multus-cni.io/default-network: kube-system/host-device-conf
```

Comment:
We also need a SpiderMultusConfig object for `host-device`, just follow

https://github.com/spidernet-io/spiderpool/blob/da3d6d9c8fd2c3a7e3d0f6ed0b9484c3dee0d643/docs/usage/install/underlay/get-started-macvlan.md?plain=1#L95-L108